client/e2e/tests/EventsPage.spec.ts

```diff
@@ -2,75 +2,109 @@
 import { test } from '@playwright/test';
 import { EventsPage } from '../pages/EventsPage';
 import { createUniqueEvent } from '../testData/factoryFunctions';
-import { EVENT_INFO_2, EVENT_INFO_3, EVENT_INFO_4 } from '../testData/dataConstants/EventInfoConstants';
+import { EVENT_INFO_1, EVENT_INFO_2, EVENT_INFO_3, EVENT_INFO_4 } from '../testData/dataConstants/EventInfoConstants';
 import { SHOWING_INFO_1, SHOWING_INFO_2, SHOWING_INFO_3, SHOWING_INFO_4 } from '../testData/dataConstants/ShowingInfoConstants';
 
 test('Homepage->Events', async ({page}) => {
   const eventsPage = new EventsPage(page);
   await eventsPage.goto();
 });
 
+test('addNewEvents', async ({page})=>{
+  const eventsPage = new EventsPage(page);
+  const currentEvent =  createUniqueEvent(EVENT_INFO_1);
+  await eventsPage.goto();
+  await eventsPage.addnewevent(currentEvent);
+  await eventsPage.activeEvent();
+  await eventsPage.deleteTheEvent(currentEvent.eventFullName);
+});
 
-test.skip('addDeleteEvents', async ({page})=>{
+/**
+ * First we create a new event and add a new showing for it.
+ * And we go back to homepage to see if it exists on the homepage.
+ * Go to the newly created event from the manage ticketing page.
+ * Add one more showing for it.
+ * Search for two corresponding showing by their date and delete them.
+ * Delete the newly created event at last.
+ */
+test('addDeleteEvents', async ({page})=>{
   // test.setTimeout(300000);
   const eventsPage = new EventsPage(page);
   const currentEvent = createUniqueEvent(EVENT_INFO_2);
   await eventsPage.goto();
-  // The ANE_Package2 is locate in ConstsPackage.ts file
-  // First we create a new event
   await eventsPage.addnewevent(currentEvent);
-  await eventsPage.activateEvent();
-  // Then we add a new showing for it
+  await eventsPage.activeEvent();
   await eventsPage.addNewShowing(SHOWING_INFO_1);
-  // Go back to homepage to see if it exists on the homepage
-  await eventsPage.checkNewEventOnHomePage();
-  // Go to the newly created event from the manage ticketing page
+  await eventsPage.checkNewEventOnHomePage(currentEvent.eventName,currentEvent.suffix);
   await eventsPage.goToEventFromManage(currentEvent.eventFullName);

```

Comment:
you use the eventFullName here, you can generate this within the goToEventFromManage Function itself.  Just pass it the entire Event. That way you have both name and description to use.